### PR TITLE
update faq to current client information

### PR DESCRIPTION
**Pull request by KolbyML**

### CODE REVIEW

**Review by kdeme — APPROVED** [review]

File: pages/resources/faq.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ Other networks are in earlier development and being actively worked on by the Po
 We intend to develop all five subprotocols into production-ready public networks, starting with the History Network. The State network is the next priority so that we can support light clients. Building out this core infrastructure is our primary goal. We believe many use cases will emerge when the infrastructure is in place and we anticipate working with Ethereum client developers and application developers wanting to develop truly decentralized backends for their products.
 
 
-## Why develop three clients in parallel?
+## Why develop four clients in parallel?
```
Comment (njgheorghita) [inline pages/resources/faq.md]: `Why develop multiple clients in parallel?`

**Review by njgheorghita — DISMISSED** [review]

File: pages/resources/faq.md (hunk 1 above)
Comment (KolbyML) [reply pages/resources/faq.md]: I don't have permissions to merge so if my fixup PR looks good can you do it :shipit:. I believe I resolved this concern, but if I didn't let me know and I can take another shot at it

File: pages/resources/faq.md [hunk 2]
```diff
@@ -24,9 +24,9 @@ Other networks are in earlier development and being actively worked on by the Po
 We intend to develop all five subprotocols into production-ready public networks, starting with the History Network. The State network is the next priority so that we can support light clients. Building out this core infrastructure is our primary goal. We believe many use cases will emerge when the infrastructure is in place and we anticipate working with Ethereum client developers and application developers wanting to develop truly decentralized backends for their products.
 
 
-## Why develop four clients in parallel?
+## Why develop mutiple clients in parallel?
```
Comment (njgheorghita) [inline pages/resources/faq.md]: `multiple` not `mutiple` for all these changes

Comment (KolbyML) [reply pages/resources/faq.md]: should be good now

**Review by njgheorghita — APPROVED** [review]

**Pull request merged**